content/en/docs/reference/command-line-tools-reference/feature-gates/ComponentSLIs.md

```diff
@@ -13,6 +13,12 @@ stages:
   - stage: beta
     defaultValue: true
     fromVersion: "1.27"
+    toVersion: "1.31"
+  - stage: stable
+    locked: true
```

Comment:
This is OK, but maybe slightly misleading; we don't support a `locked` key in anything I know of.

Reply:
`locked` is gonna be important, because a feature which has `lockedToDefault: true` will not be shown from the help message such as `kube-apiserver --help`.
The gate is there, not removed. You cannot change its setting. A weird state it is.
We will need to revise the shortcode someday to reflect this.
Adding `locked: true` is a first step. At least, it is doing no harm at the moment.

Reply:
Filed one: https://github.com/kubernetes/website/issues/49388